en/test_and_ci/unit_tests.md

```diff
@@ -0,0 +1,128 @@
+# Unit Tests
+
+PX4 provides a simple base [Unittest-class](https://github.com/PX4/Firmware/blob/master/src/include/unit_test.h). Each developer is encouraged to write unit tests in the process of adding a new feature to the PX4 framework.
+
+## Writing a Test
+
+1. Create a new .cpp file within [tests](https://github.com/PX4/Firmware/tree/master/src/systemcmds/tests) with name `test_[description].cpp`. 
+
+2. Within `test_[description].cpp` include the base unittest-class `<unit_test.h>` and all files required to write a test for the new feature. 
+
+3. Within `test_[description].cpp` create a class `[Description]Test` that inherits from `UnitTest`.
```

Comment:
I tend to use bold for filenames.

Reply:
sure. however, that is what currently is used within px4. 
https://github.com/PX4/Firmware/tree/master/src/systemcmds/tests
I think until we have not decided on a guideline here https://github.com/PX4/Firmware/issues/8319, we should keep it the way it is.

Reply:
I think we might be confused. I mean, the above text should render in gitbook as:

3. Within **test_[description].cpp** create a class `[Description]Test` that inherits from `UnitTest`.

rather than what it is currently:

3. Within `test_[description].cpp` create a class `[Description]Test` that inherits from `UnitTest`.

Reply:
ah alright, my bad. sure, that makes sense.